### ISSUE TITLE
TFSlim trained 'mobilenetv2' model is giving poor accuracy with is_training='False' (When save and restore the checkpoints)

### ISSUE THREAD

**Issue by joyalbin** [tensorflow/tensorflow#30274]
#### Run training.
python train_image_classifier.py \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=cifar10 \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=mobilenet_v2 \
  --preprocessing_name=mobilenet_v2 \
  --max_number_of_steps=100000 \
  --batch_size=48 \
  --save_interval_secs=120 \
  --save_summaries_secs=120 \
  --log_every_n_steps=100 \
  --optimizer=adagrad \
  --learning_rate=0.1 \
  --learning_rate_decay_factor=0.1 \
  --num_epochs_per_decay=200 \
  --weight_decay=0.004 \
  --moving_average_decay=0.9999

#### Run evaluation.
python eval_image_classifier.py \
  --checkpoint_path=${TRAIN_DIR} \
  --eval_dir=${TRAIN_DIR} \
  --dataset_name=cifar10 \
  --dataset_split_name=test \
  --dataset_dir=${DATASET_DIR} \
  --model_name=mobilenet_v2

### 3. I have saved the checkpoints with `is_training=False` from the `eval_image_classifier.py` itself. Converted the checkpoints to .pb file and then measured the evaluation accuracy. The accuracy is very less.

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 18.04
- TensorFlow installed from  binary
- TensorFlow version (use command below):  1.13
- Python version: 3.7.3
- Bazel version (if compiling from source): 24.1
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the current behavior**
The modem accuracy changed after saving the evaluation graph with `'is_training=False'`

**Describe the expected behavior**
The model should give same accuracy after saving with `'is_training=False'`

**Code to reproduce the issue**
Training with slim framework

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by sguada** [comment]
Given `--max_number_of_steps=100000` you should probably lower the `--moving_average_decay=0.9999` to `--moving_average_decay=0.99` to make sure the moving average are updated.

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
